osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/WatchHighScoresCommand.cs

```diff
@@ -123,9 +123,9 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                 while (!cancellationToken.IsCancellationRequested)
                 {
                     HighScore[] highScores = (await dbMainQuery.QueryAsync<HighScore>(
-                                                 "SELECT h.*, s.id as new_id FROM osu.score_process_queue "
+                                                 "SELECT q.*, h.*, s.id as new_id FROM osu.score_process_queue q "
```

Comment:
This is a bit of cheating: we want to get `score_id` from the queue table (because it will be null in a deletion from the `highScoreTable`). Dapper will use the first available, which becomes the id from the queue.

Reply:
that makes me uncomfortable but not deeply enough to make a big deal out of it. let's see if i'm wrong about that